mitodl/mitxpro
# style: add trademark logos

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5719

### Description (What does it do?)
- Adds the trademark logos

### Screenshots (if appropriate):

**Certificate (Desktop)**

<img width="1792" alt="image" src="https://github.com/user-attachments/assets/530e3193-aba2-4158-838f-685499dc682b">

**Certificate (Mobile)**
<img width="216" alt="image" src="https://github.com/user-attachments/assets/3939a477-3816-4eb1-83f5-ec8f9f1dd0c7">


**Dashboard (Desktop)**
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/6592cd61-05fe-4d85-a60b-3e8b06c8f1f0">

Dashboard **(Mobile)**
<img width="216" alt="image" src="https://github.com/user-attachments/assets/64c48ac0-fa94-4cba-b353-277eeeee4f89">

**Emails**
<img width="1408" alt="image" src="https://github.com/user-attachments/assets/2551593e-ed3f-4c30-a53e-fa31577dde5c">


### How can this be tested?
- Setup xPRO on this branch
- Verify that the application shows trademark logos everywhere e.g. Signup/Login, Dashboard, Course Page, Registration/enrollment Emails
